Gifski/Gifski.swift

```diff
@@ -253,7 +253,7 @@ final class Gifski {
 							bytesPerRow: UInt32(image.bytesPerRow),
 							height: UInt32(image.height),
 							pixels: buffer,
-							presentationTimestamp: result.actualTime.seconds
+							presentationTimestamp: max(0, result.actualTime.seconds - startTime)
```

Comment:
@kornelski `result.actualTime.seconds - startTime` is often a negative value as the actual time might be before our requested time, hence before the startTime. Does libgifski handle negative PTS? Should I just pass it negative to libgifski or is it correct to handle it here? If passing it to libgifski works, I assume it will just throw it away, but wouldn't that mean, we'll lose a frame?

Example value: `-0.012468888888889751`.

Reply:
It doesn't handle negative PTS.